lib/prawn-fillform.rb

```diff
@@ -9,6 +9,11 @@ module Prawn
 
   module Fillform
 
+    FLAG_REQUIRED       =  2.freeze
```

Comment:
I don't know why i know this, but when the field is readonly and required it's '3'
:-(
hawaiian-ice/app/services/document/pdf_field.rb